packages/components/src/public/QueryColumn.ts

```diff
@@ -157,6 +158,7 @@ export class QueryColumn extends Record({
     declare shownInInsertView: boolean;
     declare shownInUpdateView: boolean;
     declare sortable: boolean;
+    declare filterable: boolean;
```

Comment:
It looks like `DisplayColumn` on the server has a `isFilterable()` flag. Is this something we should hook-up `filterable` to round-trip in `JsonWriter` (similar to the `sortable` property)? This looks like it is the same as is respected by LKS for determining if filtering should be available for a column.

Otherwise, if we do just leave this as a "client-side" property only then I suggest it's declaration be added to the section or properties that are only on the client. A comment is missing in this class for that but it starts with `declare cell: Function` ... could you also add a comment for that section.

Reply:
good suggestion, change made in platform to wire up filterable in JsonWriter